content/posts/2024-06-12_trunk_and_tidbits-may_2024/index.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Trunk & Tidbits, May 2024
+description: "Updates from the core Mastodon engineering team for May 2024"
+date: 2024-06-12
+draft: false
+categories:
+- Trunk and Tidbits
+- Engineering
+tags:
+- engineering
+- mastodon
+authors:
+- renchap
+- andy
+resources:
+- name: hero
+  src: hero.png
+---
+
+Welcome again to Trunk & Tidbits, our engineering updates blog series. Let us fill you in on what we’ve been focused on during this past month.
+
+## Releases you should know about
+
+We released some important [updates for the 4.2 and 4.1 releases, and nightly builds](https://mastodon.social/@MastodonEngineering/112530662239602222). They contain minor security fixes, as well as bug fixes. Thanks to [the BSI](https://www.bsi.bund.de/DE/Home/home_node.html) for sponsoring a pentest of our codebase, which helped us find and fix some of these issues.
+
+As a reminder, you can [follow @MastodonEngineering](https://mastodon.social/@MastodonEngineering) to keep track of new releases and other important updates. If you are a server administrator or you want to pay special attention to these releases, you can also choose to be notified when that account posts an update.
+
+## Code changes and work in progress
+
+We’re continuing to drive towards Mastodon 4.3, our next major release.
+
+The changes below are available to users of instances that are running a recent “nightly” version (for example, [mastodon.social](http://mastodon.social) or [mastodon.online](https://mastodon.online)), and will come to others as part of the 4.3 release.
+
+# Web app and Backend
+
+<div class="features-list">
+
+- {{< feature-label name="interface" >}} Allow admins to configure the instance favicon and logo. {{< github-pr id=30040 authors="FawazFarid" >}}
+- {{< feature-label name="new-feature" >}} Add OpenTelemetry instrumentation. This allows admins to send traces to their own OpenTelemetry server, allowing monitoring of backend errors and performance with much more details than logs - useful to spot performance issues and fix them. {{< github-pr id=30130 authors="robbkidd, julianocosta89, renchap" >}}
+- {{< feature-label name="security" >}} A pair of OAuth-related updates:
```

Comment:
Oauth changes are not really security, but maybe we need an "authentication" label here?

Reply:
Fair point, or API here?